pubspec.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   plugin_platform_interface: ^2.0.2
   # plugin_platform_interface depends on meta anyway
   meta: ^1.3.0
+  web: ^1.1.0
```

Comment:
this version requires Dart 3.4
so we need to bump this https://github.com/PostHog/posthog-flutter/blob/e44de573ea0aa2068d7c33a839d2c12130b1887c/pubspec.yaml#L10-L11
and then release the SDK as a new major version

Reply:
min flutter 3.22 https://medium.com/flutter/io24-5e211f708a37 which is not super old.
can we use an older version of `web` so we can keep a better compatibility?
like https://pub.dev/packages/web/versions/0.4.2 uses Dart 3.2, Flutter 3.16

Reply:
actually i dont see any import from `package:web`, do we actually need it?